dockstore-webservice/src/main/java/io/dockstore/webservice/core/Collection.java

```diff
@@ -199,21 +203,21 @@ public void setDescription(String description) {
         this.description = description;
     }
 
-    @JsonProperty("entries")
+    @JsonProperty
```

Comment:
The real fixes are in here, `collectionEntries` should be exported as itself, not as `entries` which clashes with the property on line 139 which is now `entryVersions` to be clear, but also because @svonworl and I ran into an issue just renaming one of them.